src/strategies/strategy-mirror-farm-base.sol

```diff
@@ -65,39 +65,39 @@ abstract contract StrategyMithFarmBase is StrategyStakingRewardsBase {
         // i.e. will be be heavily frontrunned?
         //      if so, a new strategy will be deployed.
 
-        // Collects MIS tokens
+        // Collects MIR tokens
         IStakingRewards(rewards).getReward();
-        uint256 _mis = IERC20(mis).balanceOf(address(this));
-        if (_mis > 0) {
+        uint256 _mir = IERC20(mir).balanceOf(address(this));
+        if (_mir > 0) {
             // 10% is locked up for future gov
-            uint256 _keepMIS = _mis.mul(keepMIS).div(keepMISMax);
-            IERC20(mis).safeTransfer(
+            uint256 _keepMIR = _mir.mul(keepMIR).div(keepMIRMax);
+            IERC20(mir).safeTransfer(
                 IController(controller).treasury(),
-                _keepMIS
+                _keepMIR
             );
-            _swapSushiswapWithPath(mis_usdt_path, _mis.sub(_keepMIS));
+            _swapSushiswapWithPath(mir_ust_path, _mir.sub(_keepMIR));
```

Comment:
Here you are swapping MIR for UST, just below you are swapping half of UST for MIR.
So it would make more sense to here just swap half of MIR for UST and skip the second swap.
Save some gas :)